src/server/raft_client/fresh.rs

```diff
@@ -401,60 +418,69 @@ where
     fn clean_up(&mut self, sink_err: &Option<grpcio::Error>, recv_err: &Option<grpcio::Error>) {
         error!("connection aborted"; "store_id" => self.store_id, "sink_error" => ?sink_err, "receiver_err" => ?recv_err, "addr" => %self.addr);
 
-        let router = &self.router;
-        self.buffer.clear(|msg| {
-            let _ = router.report_unreachable(msg.get_region_id(), msg.get_to_peer().get_id());
-        });
-
         if let Some(tx) = self.lifetime.take() {
             let should_fallback = [sink_err, recv_err]
                 .iter()
                 .any(|e| e.as_ref().map_or(false, super::grpc_error_is_unimplemented));
             if should_fallback {
                 // Asks backend to fallback.
                 let _ = tx.send(());
+                return;
             }
         }
+
+        let router = &self.router;
+        router.broadcast_unreachable(self.store_id);
```

Comment:
Why not `report_unreachable`? Some regions don't need to send messages to the store.

Reply:
It's the same behavior as past that broken connection will cause unreachable report to all regions.